client/src/views/settings/SettingsPage.vue

```diff
@@ -0,0 +1,186 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
```

Comment:
You have dupplicated what already exists in SettingsModal. So now if we want to add an option, it's twice the work (and increased chances that we will just forget about one of the page). This has to be re-factored.